index.html

```diff
@@ -83,5 +83,270 @@ <h1 class="text-dark hero-title">
+            />
+          </div>
+          <div class="box-content fw-600">
+            <h2 class="">Multi-Post Stories Gain + Glory</h2>
+            <div class="tags flex w-full justify-between">
+              <span>Ruby on Rails</span>
+              <span>css</span>
+              <span>JavaScript</span>
+              <span>HTML</span>
+            </div>
+            <div class="flex justify-center">
+              <button
+                class="text-center btn text-white"
+                type="button"
+              >
+                See Project
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="about-section relative flex flex-col align-items-center">
+      <div class="about-bg"></div>
+      <div class="about-container">
+        <header class="about-content fw-700 leading-4 w-full text-center mb-2">
+          <h1 class="text-dark about-title">About Me</h1>
+        </header>
+        <p class="about-desc mt-0 mb-2">
+          I can help you build a I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hesitate to contact
+          me.
+        </p>
+      </div>
+      <div class="resume">
+        <button class="btn text-white" type="button">Get My Resume</button>
+      </div>
+    </section>
+
+    <section class="info-section">
+      <div class="info-box language">
+        <div class="info-content text-dark">
+          <h2 class="text-center">Languages</h2>
+          <ul class="language-list w-full">
+            <li>Ruby on rails</li>
+            <li>Css</li>
+            <li>Javascript</li>
+            <li>Html</li>
+          </ul>
+        </div>
+      </div>
+
+      <div class="info-box framework">
+        <div class="info-content text-dark">
+          <h2 class="text-center">Frameworks</h2>
+          <ul class="framework-list framework-list-1 w-full">
+            <li>React.js</li>
+            <li>Ruby on Rails</li>
+            <li>RSpec</li>
+          </ul>
+          <ul class="framework-list framework-list-2 w-full">
+            <li>Capybara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+      </div>
+
+      <div class="info-box skill">
+        <div class="info-content text-dark">
+          <h2 class="text-center">Skills</h2>
+          <ul class="skill-list skill-list-1 w-full">
+            <li>Database Management</li>
+            <li>Version Control</li>
+          </ul>
+          <ul class="skill-list skill-list-2 w-full">
+            <li>CLI</li>
+            <li>Web Development</li>
+            <li>API Design</li>
+          </ul>
+        </div>
+      </div>
+    </section>
+
+    <!-- <section class="contact-section">
+      <div>
+        <p>
+
+        </p>
+        <div>
+
+        </div>
+      </div>
+    </section> -->
   </body>
```

Comment:
- [ ] Please make sure you remove commented code from your project to make your code clean. :pray: You should not push commented code to your repository. You can read more about it [here](https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/)